_events/2023-0718-2-9-release-meetings.markdown

```diff
@@ -1,12 +1,11 @@
 ---
 
-eventdate: 2023-07-19T15:00
-tz: UTC -8
+eventdate: 2023-07-18 15:00:00 -0700
 title: OpenSearch 2.9.0 Release Meetings - 2023-07-18
-#online: true
-#signup:
-#    url: https://www.meetup.com/opensearch/events/tvcvftyfclbcc/
-#    title: Join on Meetup
+online: true
+signup:
+  url: https://us02web.zoom.us/j/83030994833
+  title: Join on Meetup
```

Comment:
This was copied from the links at the bottom of the description to avoid a useless empty button on the side.